From: zhang jiao <zhangjiao2@cmss.chinamobile.com>
Subject: samples/cgroup: rm unused MEMCG_EVENTS macro
Date: Wed, 3 Sep 2025 15:30:59 +0800

MEMCG_EVENTS is never referenced in the code. Just remove it.

Link: https://lkml.kernel.org/r/20250903073100.2477-1-zhangjiao2@cmss.chinamobile.com
Signed-off-by: zhang jiao <zhangjiao2@cmss.chinamobile.com>
Acked-by: Shakeel Butt <shakeel.butt@linux.dev>
Cc: Johannes Weiner <hannes@cmpxchg.org>
Cc: Michal Hocko <mhocko@kernel.org>
Cc: Muchun Song <muchun.song@linux.dev>
Cc: Roman Gushchin <roman.gushchin@linux.dev>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 samples/cgroup/memcg_event_listener.c |    2 --
 1 file changed, 2 deletions(-)

--- a/samples/cgroup/memcg_event_listener.c~samples-cgroup-rm-unused-memcg_events-macro
+++ a/samples/cgroup/memcg_event_listener.c
@@ -18,8 +18,6 @@
 #include <sys/inotify.h>
 #include <unistd.h>
 
-#define MEMCG_EVENTS "memory.events"
-
 /* Size of buffer to use when reading inotify events */
 #define INOTIFY_BUFFER_SIZE 8192
 
_
